src/pages/api/stageNames.ts

```diff
@@ -13,10 +26,12 @@ export const isStageNameValid = (req: NextApiRequest): InputCheck[] => {
             error = 'Enter a name for this fare stage';
         } else if (stageNameInput[i].length > 30) {
             error = `The name for Fare Stage ${i + 1} needs to be less than 30 characters`;
+        } else if (stageNameInArrayMultipleTimes(stageNameInput, stageNameInput[i])) {
+            error = 'Stage names cannot share exact names';
```

Comment:
They shouldn't - for one, because we don't support it on the fare triangle page. If we want to support it, we can change the fare triangle page ID generation but until then, this fix stops that from breaking. Secondly, I can't see fares information being easy to consume if fare stages have the same name, making it hard to figure out prices.